public/css/employeeDetails.css

```diff
@@ -0,0 +1,219 @@
+.box .btn-special,
+.box .btn-save,
+.box .btn-report{
+  box-shadow: -1px 1px 3px 0px #201a3a;
+  float: left;
+  border: none;
+  background: #201a3a;
+  color: #ffffff;
+  border-radius: 5px;
+  padding: 5px 10px;
+  width: 100px;
+  font-weight: bold;
+  transition-duration: 0.3s;
+  cursor: pointer;
+}
+
+.none {
+  display: none !important;
+}
+
+.box .btn-save:hover {
+  background-color: #3f3663;
+}
+.box ul li select{
+  border: none;
+  border-radius: 5px;
+  box-shadow: -1px 1px 3px 0px #201a3a;
+  width: 168px;
+  padding: 0px 10px;
+  background: none;
+  height: 30px;
+}
+.btn-special {
+  background: red !important;
+  margin: 5px 10px 0;
+  width: 75px !important;
+}
+
+.btn-special:hover {
+  background: rgb(173, 63, 63) !important;
+}
+
+.box .bar ul {
+  display: flex;
+  justify-content: flex-start;
+  list-style: none;
+}
+.box .bar ul li {
```

Comment:
### indentation 
add a single white line after each style block
**Same for the rest of the file**